packages/wallet/contract/package.json

```diff
@@ -18,17 +18,27 @@
     "@agoric/inter-protocol": "^0.11.0",
     "@agoric/swingset-vat": "^0.28.0",
     "@agoric/vats": "^0.10.0",
+    "@agoric/wallet-backend": "0.12.1",
     "@endo/captp": "^2.0.13"
   },
   "dependencies": {
-    "@agoric/wallet-backend": "0.12.1",
+    "@agoric/assert": "^0.4.0",
+    "@agoric/cache": "^0.1.0",
     "@agoric/deploy-script-support": "^0.9.0",
+    "@agoric/ertp": "^0.14.2",
+    "@agoric/inter-protocol": "^0.11.0",
```

Comment:
wallet depends on inter-protocol?

Is it perhaps just for types? It could be a devdependency then, yes?

Reply:
good catch. not sure why I added this since it was already in the devDeps list